src/Lift.php

```diff
@@ -118,16 +119,15 @@ public function addLiftAccountAndSiteId()
     /**
      * Pings the service to ensure that it is available.
      *
-     * @return \Psr\Http\Message\ResponseInterface
-     *
-     * @throws \GuzzleHttp\Exception\RequestException
+     * @return mixed the value encoded in the JSON.
      */
     public function ping()
     {
-        // Now make the request.
         $request = new Request('GET', '/ping');
+        $response = $this->authenticatedClient->send($request);
+        $body = (string) $response->getBody();
 
-        return $this->getResponseJson($request);
+        return json_decode($body, true);
```

Comment:
There is a bit of duplicated code here as opposed to ManagerBase class's code, but I think it is too much to create a ping manager or move any code to a trait.